config.json

```
@@ -408,7 +408,38 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Simple",
+      "content": "Zig lets you focus on debugging your application rather than debugging your programming language knowledge.",
+      "icon": "easy"
+    },
+    {
+      "title": "Comptime",
+      "content": "Zig has a a fresh approach to metaprogramming based on compile-time code execution and lazy evaluation",
```

This line exceeds the maximum length of 100 by 2 characters. Could you shorten it?

---

```suggestion
      "content": "Zig has a fresh approach to metaprogramming based on compile-time code execution and lazy evaluation",
```

This is a huge selling point of the language.

 If anyone can wordsmith this a bit to explain it better, that'd be awesome. As-is I think most discussion around comptime has yet to really "sell" the concept.
 
Comptime means something like "You can do everything that the C preprocessor can do, but it's safe! And it's first class! And it's the same language with the same syntax with the same type discipline!" ...But also that sounds a lot like homoiconic languages like Lisps/Tcl/etc. but it's something that not just allows for metaprogramming, but allows for safe, compile-time checked metaprogramming.

Rambles aside, if no one here has a knack for marketing I'd say just remove the second `a` and move on.